activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/converter/VariableCreatedEventConverter.java

```diff
@@ -35,8 +35,8 @@ public ProcessEngineEvent from(ActivitiEvent event) {
                                             event.getProcessDefinitionId(),
                                             event.getProcessInstanceId(),
                                             ((ActivitiVariableEventImpl) event).getVariableName(),
-                                            ((ActivitiVariableEventImpl) event).getVariableValue().toString(),
-                                            ((ActivitiVariableEventImpl) event).getVariableType().getTypeName(),
+                                            String.valueOf(((ActivitiVariableEventImpl) event).getVariableValue()),
+                                            (((ActivitiVariableEventImpl) event).getVariableType() != null)?((ActivitiVariableEventImpl) event).getVariableType().getTypeName():"",
```

Comment:
Is there any specific reason for that you are converting from `null` to an empty String?

Reply:
@erdemedeiros look at my previous comment. I was trying to avoid sending Null in the json back. if the variable has no value, in my opinion it should be empty. But I don't hold any strong position on sending "null"

Reply:
I'd prefer `null` instead of empty because at the process engine side the variable value is an `Object`, so probably we don't want to convert it to string here, but let the serialisation library take care of this. What do you think?